lib/main.dart

```diff
@@ -76,11 +76,13 @@ import 'services/music_player_background_task.dart';
 import 'services/theme_mode_helper.dart';
 import 'services/ui_overlay_setter_observer.dart';
 import 'setup_logging.dart';
+import 'services/metadata_helper.dart';
 
 void main() async {
   // If the app has failed, this is set to true. If true, we don't attempt to run the main app since the error app has started.
   bool hasFailed = false;
   try {
+    await setupLocator();
```

Comment:
please either namespace this (`import 'services/metadata_helper.dart' as metadataHelper;` and then `await metadataHelper.setupLocator()`), or move the setup function to main.dart (like all the other setup functions).